sql/catalyst/src/main/java/org/apache/spark/sql/catalog/v2/IdentifierImpl.java

```diff
@@ -49,6 +49,11 @@ public String name() {
     return name;
   }
 
+  @Override
+  public String toString() {
+    return Arrays.toString(namespace) + " " + name;
```

Comment:
I think this should produce the quoted identifier that can be parsed, like what is produced by `CatalogV2Implicits.IdentifierHelper`.